src/pages/settings/Profile/ProfilePage.js

```diff
@@ -241,17 +254,36 @@ class ProfilePage extends Component {
                         <Text style={[styles.mt6, styles.mb6]}>
                             {this.props.translate('profilePage.tellUsAboutYourself')}
                         </Text>
-                        <FullNameInputRow
-                            firstName={this.state.firstName}
-                            firstNameError={PersonalDetails.getMaxCharacterError(this.state.hasFirstNameError)}
-                            lastName={this.state.lastName}
-                            lastNameError={PersonalDetails.getMaxCharacterError(this.state.hasLastNameError)}
-                            onChangeFirstName={firstName => this.setState({firstName})}
-                            onChangeLastName={lastName => this.setState({lastName})}
-                            style={[styles.mt4, styles.mb4]}
-                        />
+
+                        <View style={[styles.flexRow, styles.mt4, styles.mb4]}>
+                            <View style={styles.flex1}>
+                                <TextInput
+                                    inputID="firstName"
+                                    name="fname"
+                                    label={this.props.translate('common.firstName')}
+                                    value={this.state.firstName}
+                                    hasError={this.state.hasFirstNameError}
+                                    errorText={PersonalDetails.getMaxCharacterError(this.state.hasFirstNameError)}
+                                    onChangeText={firstName => this.setState({firstName})}
+                                    placeholder={this.props.translate('profilePage.john')}
+                                />
+                            </View>
+                            <View style={[styles.flex1, styles.ml2]}>
+                                <TextInput
+                                    inputID="lastName"
+                                    name="lname"
+                                    label={this.props.translate('common.lastName')}
+                                    value={this.state.lastName}
+                                    hasError={this.state.hasLastNameError}
+                                    errorText={PersonalDetails.getMaxCharacterError(this.state.hasLastNameError)}
+                                    onChangeText={lastName => this.setState({lastName})}
+                                    placeholder={this.props.translate('profilePage.doe')}
+                                />
+                            </View>
+                        </View>
                         <View style={styles.mb6}>
                             <Picker
+                                inputID="pronounsPicker"
```

Comment:
I think `pronouns` is a better name, since we already use this name in `updatePersonalDetails`